scripts/index.js

```diff
@@ -0,0 +1,117 @@
+const state = {
+    temp: 82,
+    sky: "Sunny",
+    city: "Emerald City"
+};
+
+const skys = [`☁️ ☁️ ☁️ ☀️ ☁️ ☁️`,
+    `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`,
+    `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`,
+    `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`
+]
+
+const landscapes = [
+    `🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂`,
+    `🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷`,
+    `🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃`,
+    `🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲`
+]
+
+const incrementTemp = () => {
+    let temp = document.getElementById("temp");
+    state.temp += 1;
+    temp.innerHTML = state.temp;
+    const landscapeIdx = getLandscapeIdx()
+
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const decrementTemp = () => {
+    const temp = document.getElementById("temp");
+    state.temp -= 1;
+    temp.innerHTML = state.temp;
+
+    const landscapeIdx = getLandscapeIdx()
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const getLandscapeIdx = () => {
```

Comment:
Great idea to have a helper like this, but there's no intrinsic relationship between the array of values and these ranges, so it would be possible to end up having this code and the array getting out of sync. Consider a more data-driven approach that encodes the date ranges in the data itself.